include/seqan3/alphabet/aminoacid/concept.hpp

```diff
@@ -48,54 +48,32 @@
 namespace seqan3
 {
 
+//!\brief Helper metafunction that identifies amino acid alphabets.
+//!\ingroup aminoacid
```

Comment:
I would add something like this to \details

### Customisation point

If you define your own alphabet and want it to be recognised as an amino acid alphabet by SeqAn, you need to specialise this metafunction for your type and have it inherit std::true_type. 

```cpp
namespace your_namespace
{

struct your_aa
{
//...
};

}

namespace seqan3
{

template <>
struct is_aminoacid<your_namespace::your_aa> : std::true_type {};

}
```